API/TrelloHelper/TrelloHelper/Startup.cs

```diff
@@ -31,7 +31,7 @@ public void ConfigureServices(IServiceCollection services)
 			// TODO: move this somewhere
             services.AddHttpClient<TrelloHttpClient>(client =>
```

Comment:
Register `TrelloClient` as interface implementation `ITrelloClient` and remove `TrelloHttpClient`